### PR TITLE
fix: reduce space between auction lot buttons

### PR DESCRIPTION
**Pull request by olerichter00**
The type of this PR is: **Bugfix**

This PR resolves [CX-1902]

### Description

Reduces space between auction lot buttons.

| Before | After |
|---|---|
| simulator_screenshot_143C3F66-9776-435D-B077-AC28CAAF70C8 | simulator_screenshot_EF687658-83AE-48F1-A8D3-6919084BA98E |

### PR Checklist (tick all before merging)

- [x] I have included screenshots or videos to illustrate my changes, or I have not changed anything that impacts the UI.
- [x] I have tested my changes on **iOS** and **Android**.
- [x] I have added tests for my changes, or my changes don't require testing, or I have included a link to a separate Jira ticket covering the tests.
- [x] I have added a feature flag, or my changes don't require a feature flag. ([How do I add one?](https://github.com/artsy/eigen/blob/master/docs/developing_a_feature.md))
- [x] I have documented any follow-up work that this PR will require, or it does not require any.
- [x] I have added an app state migration, or my changes do not require one. ([What are migrations?](https://github.com/artsy/eigen/blob/master/docs/adding_state_migrations.md))
- [x] I have added a changelog entry below or my changes do not require one.

<details><summary>Changelog updates</summary>

#### Cross-platform user-facing changes

- Fix auction lot action button spacing - ole

#### iOS user-facing changes

-

#### Android user-facing changes

-

#### Dev changes

-

</details>

[CX-1902]: https://artsyproduct.atlassian.net/browse/CX-1902

### CODE REVIEW

**Comment by ArtsyOpenSource** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Warnings</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:warning:</td>
      <td>

  ❌ **An error occurred while validating your changelog, please make sure you provided a valid changelog.**
  </td>
    </tr>
  </tbody>
</table>



<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against dd2bea2b97502b392e0496e9fbfe6669f2912ef2
</p>

**Comment by MounirDhahri** [comment]
I left a question, but please feel free to merge

File: Artsy.xcodeproj/project.pbxproj [hunk 1]
```diff
@@ -3276,6 +3276,8 @@
 				"${PODS_ROOT}/Artsy+UIFonts/Pod/Assets/Unica77LL-Regular.otf",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black.png",
 				"${PODS_ROOT}/Artsy+UILabels/Pod/Assets/Chevron_Black@2x.png",
+				"${PODS_ROOT}/../emission/Pod/Assets/Emission.js",
+				"${PODS_ROOT}/../emission/Pod/Assets/assets",
```
[the remark this answers is not in the document]
Comment (olerichter00) [reply Artsy.xcodeproj/project.pbxproj:3280]: Running `bundle exec pod install` changed the file. Do you know exactly what could have caused it?

[the remark this answers is not in the document]
Comment (MounirDhahri) [reply Artsy.xcodeproj/project.pbxproj:3280]: no idea. I thought emission should be already linked to the xcodeproject

**Pull request merged**